modules/op-installing-pipelines-operator-using-the-cli.adoc

```diff
@@ -27,7 +27,7 @@ spec:
   source: redhat-operators <3>
```

Comment:
from:
`You can install {pipelines-title} Operator from the OperatorHub using the CLI.`
to:
`You can install {pipelines-title} Operator from the OperatorHub by using the CLI.`